### PR TITLE
Transaction List & Transaction Details spec

### PR DESCRIPTION
**Pull request by tschubotz**
Spec for #71 

- [x] Use case scenarios
- [x] User interfaces

### CODE REVIEW

File: specs/assets_overview/02_user_interface.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+2. Transction details - Outgoing ETH
+------------------------------------
+
+2.1. Transction details - Failed
+--------------------------------
+
+2.2. Transction details - Pending
+---------------------------------
+
+2.3. Transction details - Outgoing ETH
```
Comment (rmeissner) [inline specs/assets_overview/02_user_interface.rst]: this is a little confusing because the title of `2` is the same

Comment (tschubotz) [reply specs/assets_overview/02_user_interface.rst]: yeah, it should say "token" 😇

File: specs/transaction_overview/02_user_interface.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+Transaction list
+------------------
+
+**Sections:**
+
+- Pending: All pending transactions
+- Today: All transactions of today
+- Yesterday: All transactions of yesterday
+- Please put all older transactions in a section that uses the date in the section title.
```
Comment (sche) [inline specs/transaction_overview/02_user_interface.rst]: IMO in majority cases there will be not more than one transaction per day. Transaction date will be duplicated in two places: section header & transaction cell. Can we use just "Pending", "Today", "Yesterday" and "Past"? Now on iOS we have "Pending", "Today" and "Past" only.

**Comment by tschubotz** [comment]
> How should this features behave without internet?

If there are still transactions cached, then display them, if not then show an empty list?
What would you suggest?

Also, we should probably have a similar solution to #125

File: specs/transaction_overview/02_user_interface.rst (hunk 2 above)
Comment (tschubotz) [reply specs/transaction_overview/02_user_interface.rst]: > Transaction date will be duplicated in two places: section header & transaction cell. 

The current designs just have the time in the cell. So there is no duplication :)

**Comment by sche** [comment]
Right now there is no transactions sync with service, so no internet should not be an issue. Or do you mean pending transaction going to Today @rmeissner ?

**Pull request merged**